tokenid_test.go

```diff
@@ -0,0 +1,15 @@
+package ens
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestDeriveTokenIdFromENSName(t *testing.T) {
```

Comment:
Please could we add some tests that fail?  Empty domain, invalid domain, that type of thing.